server/vulnerabilities/oval/input/dpkg_state.go

```diff
@@ -1,18 +0,0 @@
-package oval_input
-
-type ovalSimpleTypeXML struct {
-	Datatype string `xml:"datatype"`
-	Value    string `xml:",chardata"`
-	Op       string `xml:"operation,attr"`
-}
-
-// DpkgStateXML see https://oval.mitre.org/language/version5.10.1/ovaldefinition/documentation/linux-definitions-schema.html#dpkginfo_state.
-type DpkgStateXML struct {
```

Comment:
Renamed to `DpkgInfoStateXML` - so that the naming we use follows the spec as a close as possible.